packages/client/src/components/project/RenameDatasetDialog.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from "react";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, TextField } from "@material-ui/core";
+import gql from "graphql-tag";
+import { useMutation } from "@apollo/client";
+
+const RENAME_PROJECT_MUTATION = gql`
+  mutation RenameProject($id: ID!, $newName: String!) {
+    renameProject(id: $id, newName: $newName) {
+      id
+      name
+    }
+  }
+`;
+
+export default function RenameProjectDialog(props: { id: string }): ReactElement {
```

Comment:
Could you rename this file to RenameProjectDialog.tsx?

Reply:
yes